[QE] fix oc rollout failure in test

>Step executing "oc rollout status -w dc/quarkus --timeout=600s" succeeds: command 'oc rollout status -w dc/quarkus --timeout=600s', expected to succeed, exited with exit code: 1
Command stdout:
Command stderr: Warning: apps.openshift.io/v1 DeploymentConfig is deprecated in v4.14+, unavailable in v4.10000+
Error from server (NotFound): deploymentconfigs.apps.openshift.io "quarkus" not found


>Step executing "oc rollout status deployment hello" succeeds: command 'oc rollout status deployment hello', expected to succeed, exited with exit code: 1
Command stdout: Waiting for deployment "hello" rollout to finish: 0 of 1 updated replicas are available...
Command stderr: error: deployment "hello" exceeded its progress deadline

another log:
>02:09:36.464090  stdout: [INFO] oc: Creating BuildServiceConfig quarkus-s2i for Source build
02:09:36.482063  stdout: [INFO] oc: Creating ImageStream quarkus
02:09:36.943796  stdout: [INFO] oc: Starting Build quarkus-s2i
02:09:42.118217  stdout: [INFO] oc: Waiting for build quarkus-s2i-1 to complete...
02:09:42.172341  stdout: [INFO] oc: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"builds.build.openshift.io \"quarkus-s2i-1\" is forbidden: User \"system:serviceaccount:testproj:jkube-application-deploy-sa\" cannot get resource \"builds/log\" in API group \"build.openshift.io\" in the namespace \"testproj\"","reason":"Forbidden","details":{"name":"quarkus-s2i-1","group":"build.openshift.io","kind":"builds"},"code":403}
02:09:58.197836  stdout: [ERROR] oc: Failed to execute the build [Unable to build the image using the OpenShift build service]
02:09:58.198031  stdout: [INFO] ------------------------------------------------------------------------
02:09:58.198052  stdout: [INFO] BUILD FAILURE
02:09:58.198060  stdout: [INFO] ------------------------------------------------------------------------
02:09:58.198067  stdout: [INFO] Total time:  01:26 min
02:09:58.198076  stdout: [INFO] Finished at: 2026-01-22T01:09:58Z
02:09:58.198083  stdout: [INFO] ------------------------------------------------------------------------
02:09:58.198091  stdout: [ERROR] Failed to execute goal org.eclipse.jkube:openshift-maven-plugin:1.18.2:build (default-cli) on project quarkus: Failed to execute the build: Unable to build the image using the OpenShift build service: OpenShift Build quarkus-s2i-1 failed: Failed: BuildPodEvicted -> [Help 1]
02:09:58.198098  stdout: [ERROR] 
02:09:58.198105  stdout: [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
02:09:58.198111  stdout: [ERROR] Re-run Maven using the -X switch to enable full debug logging.
02:09:58.198118  stdout: [ERROR] 
02:09:58.198124  stdout: [ERROR] For more information about the errors and possible solutions, please read the following articles:
02:09:58.198131  stdout: [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
02:09:59.201604  stdout: exitCodeOfLastCommandInShell=0
02:09:59.201759    bash: oc rollout status -w dc/quarkus --timeout=600s
02:09:59.352590  stderr: Warning: apps.openshift.io/v1 DeploymentConfig is deprecated in v4.14+, unavailable in v4.10000+
02:09:59.355317  stderr: Error from server (NotFound): deploymentconfigs.apps.openshift.io "quarkus" not found
02:09:59.358579  stdout: exitCodeOfLastCommandInShell=1

--- a/test/e2e/features/application_deployment.feature
+++ b/test/e2e/features/application_deployment.feature
@@ -5,6 +5,7 @@ Feature: Application Deployment Test
     deployed successfully and is accessible via route
 
     Background:
+        Given setting config property "disk-size" to value "40" succeeds
         Given ensuring CRC cluster is running
         And ensuring oc command is available
         And ensuring user is logged in succeeds

